tyr/servers/nginx/server.py

```diff
@@ -35,5 +38,19 @@ def __init__(self, group=None, server_type=None, instance_type=None,
                                           chef_path, subnet_id, dns_zones)
 
     def configure(self):
-
+        super(NginxServer, self).establish_logger()
```

Comment:
Sure thing, I just saw that this was how it was called from iis/node.py and followed the same line